deploy/docs/Best_Practices.md

```diff
@@ -103,6 +104,42 @@ to the `Docker_Mode_Parser` parameter in the `Input plugin` configuration of flu
 Docker_Mode_Parser multi_line
 ```
 
+## Choosing Fluentd Base Image
+
+Historically, the Fluentd container image used with the collection was based on Debian Linux distribution.
+
+Currently, an Alpine-based image is also available and can be used instead of the Debian-based image.
+
+The Debian-based image is the default, so you do not need to change anything to use it.
+
+To use an Alpine-based image with the collection, specify an Alpine image's tag in `fluentd.image.tag` chart property:
+
+```yaml
+fluentd:
+  image:
+    tag: <Fluentd-release>-alpine
+```
+
+For example:
+
+```yaml
+fluentd:
+  image:
+    tag: 1.12.2-sumo-4-alpine
+```
+
+Go to the [official Fluentd image repository](https://gallery.ecr.aws/sumologic/kubernetes-fluentd)
+to find the latest release of Fluentd.
+The Alpine-based releases are the ones with the `-alpine` suffix.
+
+Both Debian-based and Alpine-based images support the same architectures:
+
+- x86-64,
+- ARM 32-bit,
+- ARM 64-bit.
+
+The source code and the `Dockerfile`s for both images can be found at https://github.com/SumoLogic/sumologic-kubernetes-fluentd.
```

Comment:
```suggestion
The source code and the `Dockerfile`s for both images can be found at [https://github.com/SumoLogic/sumologic-kubernetes-fluentd](https://github.com/SumoLogic/sumologic-kubernetes-fluentd).
```

Reply:
Oh I don't think that's needed when the URL and the text are the same. GitHub (and I think any Markdown viewer) shows this as a link anyway.

Reply:
As long as mdl is happy I'm fine :)